test/python/circuit/test_controlled_gate.py

```diff
@@ -507,89 +507,37 @@ def test_multi_controlled_u1_matrix(self, num_controls):
             with self.subTest(msg=f"control state = {ctrl_state}"):
                 self.assertTrue(matrix_equal(simulated, expected))
 
-    @data(1, 2, 3, 4)
-    def test_multi_control_toffoli_matrix_clean_ancillas(self, num_controls):
-        """Test the multi-control Toffoli gate with clean ancillas.
```

Comment:
Do I understand correctly that this and other tests were removed because the new `test_multi_control_toffoli_matrix_advanced_num_ancillas` test handles all of the different cases?

Reply:
yes